pkgs/tools/filesystems/xfs_undelete/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper
+, coreutils, tcl-8_6, tcllib
+}:
+
+stdenv.mkDerivation  rec {
```

I don't think the `rec` is needed here. Also, there's a double space

```suggestion
stdenv.mkDerivation {
```